frontend/packages/yki/src/components/registration/PublicIdentificationGrid.tsx

```diff
@@ -1,64 +1,187 @@
+    (event: ChangeEvent<HTMLInputElement>) => {
+      if (fieldErrors[fieldName]) {
+        handleErrors(fieldName)(event);
+      }
+
+      setEmail(event.target.value);
+    };
+
+  const showCustomTextFieldError = (
+    fieldName: keyof Pick<PublicSuomiFiRegistration, 'email'>
+  ) => {
+    return fieldErrors[fieldName].length > 0;
+  };
+
   if (!examSession) {
     return null;
   }
 
   const renderDesktopView = () => (
     <>
       <Grid className="public-registration" item>
-        <Paper elevation={3}>
-          <div className="public-registration__grid__form-container">
+        <div className="public-registration__grid">
+          <div className="rows gapped-xxl">
             <PublicRegistrationStepper activeStep={activeStep} />
-            <PublicRegistrationExamSessionDetails
-              examSession={examSession}
-              showOpenings={true}
-            />
-            <div className="margin-top-xxl gapped rows">
-              <H3>{t('title')}</H3>
-              <CustomButton
-                sx={{ width: '168px' }}
-                variant={Variant.Contained}
-                color={Color.Secondary}
-                onClick={() => {
-                  // TODO: init authentication for suomi.fi
-                  dispatch(increaseActiveStep());
-                  navigate(
-                    AppRoutes.ExamSessionRegistration.replace(
-                      /:examSessionId$/,
-                      `${examSession.id}`
-                    )
-                  );
-                }}
-                data-testid="public-registration__identify-button"
-                disabled={false}
-              >
-                {t('buttonText')}
-              </CustomButton>
+            <div className="rows">
+              <H1>{t('header')}</H1>
+              <HeaderSeparator />
             </div>
           </div>
-        </Paper>
+          <Paper elevation={3}>
+            <div className="public-registration__grid__form-container">
+              <div className="rows gapped">
+                <PublicRegistrationExamSessionDetails
+                  examSession={examSession}
+                  showOpenings={true}
+                />
+                <Text>
+                  {t('steps.identify.instructionDescription')}
+                  <ul>
+                    <li>
+                      <Text>{t('steps.identify.instructions.location')}</Text>
+                    </li>
+                    <li>
+                      <Text>{t('steps.identify.instructions.time')}</Text>
+                    </li>
+                  </ul>
+                </Text>
+
+                <div className="gapped rows">
+                  <H2>{t('steps.identify.title')}</H2>
+                  <Text>
+                    <Trans t={t} i18nKey={'steps.identify.withFinnishSSN'} />
+                  </Text>
+                  <CustomButton
+                    size="large"
+                    sx={{ width: '30rem', padding: '15px 22px' }}
```

Comment:
Nää tyylit voisi varmaan tuoda CSS-luokan kautta. Paddingit voisi varmaan olla `1.5rem 2.2rem` niin olisi ainakin teoriassa responsiivisempaa.

Reply:
Pohdin tätä kun tässä on design näkökulmasta riippuvuus tuohon materialUI:n input fieldiin ja mui input:ssa on px:nä määritelty noi paddingit ni tuntu paremmalta antaa sitten napille samat skaalaus ja pixel-to-pixel vastaavuus. Mut siirrän css-luokkiin nää ->